gulpfile.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+  'use strict';
+
+  var gulp = require('gulp'),
+      jscs = require('gulp-jscs'),
+      jshint = require('gulp-jshint'),
```

Comment:
```jscs``` is for code style and ```jshint```tries to prevent possible errors in the code, so it's ok to have both. Either way, I'm gonna investigate a little ```eslint``` and if I feel confortable using it and covers this configurations I'll use it.